src/main/java/com/example/bandservice/service/impl/BandServiceImpl.java

```diff
@@ -1,20 +1,33 @@
 package com.example.bandservice.service.impl;
 
+import com.example.bandservice.configuration.BandClientProperties;
+import com.example.bandservice.exception.NullBandReferenceException;
 import com.example.bandservice.model.Band;
+import com.example.bandservice.model.Task;
+import com.example.bandservice.model.User;
+import com.example.bandservice.model.Weapon;
 import com.example.bandservice.repository.BandRepository;
 import com.example.bandservice.service.BandService;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
 import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
 
-import java.util.List;
-import java.util.NoSuchElementException;
+import java.util.*;
```

Comment:
Is this import necessary? It is better to import every package separately if I'm not mistaken.